src/server.c

```diff
@@ -5283,6 +5283,17 @@ int processCommand(client *c) {
         return C_OK;
     }
 
+    /* check if DEBUG/Module command not allowed */
+    if ((server.debug_cmd_disabled && c->cmd->proc == debugCommand) ||
+        (server.module_cmd_disabled && c->cmd->flags & CMD_MODULE)) {
+        rejectCommandFormat(c,"%s commands not allowed, you may enable it by "
+                              "editing the Redis configuration file, and setting the "
+                              "%s option to 'no', and then restarting the server.",
+                              c->cmd->proc == debugCommand ? "DEBUG" : "Module",
```

Comment:
command names are always show in uppercase
```suggestion
                              c->cmd->proc == debugCommand ? "DEBUG" : "MODULE",
```